keyboards/naked64/info.json

```diff
@@ -0,0 +1,77 @@
+                {"label":"4", "x":6, "y":0},
+                {"label":"5", "x":7, "y":0},
+                {"label":"6", "x":9.25, "y":0},
+                {"label":"7", "x":10.25, "y":0},
+                {"label":"8", "x":11.25, "y":0},
+                {"label":"9", "x":12.25, "y":0},
+                {"label":"0", "x":13.5, "y":0},
+                {"label":"-", "x":14.5, "y":0},
+                {"label":"Back Space", "x":15.5, "y":0},
+                {"label":"pgdn", "x":0, "y":1},
+                {"label":"Tab", "x":1.25, "y":1, "w":1.5},
+                {"label":"Q", "x":2.75, "y":1},
+                {"label":"W", "x":4, "y":1},
+                {"label":"E", "x":5, "y":1},
+                {"label":"R", "x":6, "y":1},
+                {"label":"T", "x":7, "y":1},
+                {"label":"Y", "x":9.25, "y":1},
+                {"label":"U", "x":10.25, "y":1},
+                {"label":"I", "x":11.25, "y":1},
+                {"label":"O", "x":12.25, "y":1},
+                {"label":"P", "x":13.5, "y":1},
+                {"label":"[", "x":14.5, "y":1},
+                {"label":"]", "x":15.5, "y":1},
+                {"label":"Control", "x":1, "y":2, "w":1.75},
+                {"label":"A", "x":2.75, "y":2},
+                {"label":"S", "x":4, "y":2},
+                {"label":"D", "x":5, "y":2},
+                {"label":"F", "x":6, "y":2},
+                {"label":"G", "x":7, "y":2},
+                {"label":"H", "x":9.25, "y":2},
+                {"label":"J", "x":10.25, "y":2},
+                {"label":"K", "x":11.25, "y":2},
+                {"label":"L", "x":12.25, "y":2},
+                {"label":";", "x":13.5, "y":2},
+                {"label":"'", "x":14.5, "y":2},
+                {"label":"Return", "x":15.5, "y":2},
+                {"label":"Shift", "x":0.5, "y":3, "w":2.25},
+                {"label":"Z", "x":2.75, "y":3},
+                {"label":"X", "x":4, "y":3},
+                {"label":"C", "x":5, "y":3},
+                {"label":"V", "x":6, "y":3},
+                {"label":"B", "x":7, "y":3},
+                {"label":"N", "x":9.25, "y":3},
+                {"label":"M", "x":10.25, "y":3},
+                {"label":",", "x":11.25, "y":3},
+                {"label":".", "x":12.25, "y":3},
+                {"label":"/", "x":13.5, "y":3},
+                {"label":"&uarr;", "x":14.5, "y":3},
+                {"label":"Shift", "x":15.5, "y":3},
+                {"label":"Adjust", "x":1.25, "y":4, "w":1.75},
+                {"label":"&larr;", "x":13.5, "y":4},
+                {"label":"&darr;", "x":14.5, "y":4},
+                {"label":"&rarr;", "x":15.5, "y":4},
+                {"label":"Alt", "x":3.75, "y":4.25},
+                {"x":4.75, "y":4.25, "w":2.25},
+                {"label":"&dArr;", "x":7, "y":4.25},
+                {"label":"&uArr;", "x":9.25, "y":4.25},
+                {"x":10.25, "y":4.25, "w":2},
+                {"label":"GUI", "x":12.25, "y":4.25}
```

Comment:
Your keys are out of sequence here due to the vertical offset on the bottom row, but the positioning overall is good.

Change Lines 65-73 to the following to fix it. It just puts the keys in the correct sequence.

```json
                {"label":"Alt", "x":3.75, "y":4.25},
                {"x":4.75, "y":4.25, "w":2.25},
                {"label":"&dArr;", "x":7, "y":4.25},
                {"label":"&uArr;", "x":9.25, "y":4.25},
                {"x":10.25, "y":4.25, "w":2},
                {"label":"GUI", "x":12.25, "y":4.25},
                {"label":"&larr;", "x":13.5, "y":4},
                {"label":"&darr;", "x":14.5, "y":4},
                {"label":"&rarr;", "x":15.5, "y":4}
```

Reply:
Thank you. I fixed it.